python_modules/libraries/dagster-dbt/dagster_dbt/utils.py

```diff
@@ -40,6 +41,23 @@ def __getattr__(self, item):
             # allow recursive access e.g. foo.bar.baz
             return _DictShim(ret) if isinstance(ret, dict) else ret
 
+    if dbt_version >= "1.8.0":
+        from dbt.contracts.graph.nodes import UnitTestDefinition
+
+        unit_test = (
+            {
+                "unit_tests": {
+                    # unit test nodes must be of type UnitTestDefinition
+                    unique_id: UnitTestDefinition(**info)
+                    for unique_id, info in manifest_json["unit_tests"].items()
+                },
+            }
+            if manifest_json.get("unit_tests")
+            else {}
+        )
+    else:
+        unit_test = {}
```

Comment:
nit: remove `else`, use `version.parse`, rename to `unit_tests`

```suggestion
    unit_tests = {}
    if version.parse(dbt_version) >= version.parse("1.8.0"):
        from dbt.contracts.graph.nodes import UnitTestDefinition

        unit_tests = (
            {
                "unit_tests": {
                    # unit test nodes must be of type UnitTestDefinition
                    unique_id: UnitTestDefinition(**unit_test_resource_props)
                    for unique_id, unit_test_resource_props in manifest_json["unit_tests"].items()
                },
            }
            if manifest_json.get("unit_tests")
            else {}
        )
```

Reply:
Done in [88e20dd](https://github.com/dagster-io/dagster/pull/23232/commits/88e20ddec7762b6e75ad918ef9ec7b02cb4c2f09)